run_dir/design/flowcells.html

```diff
@@ -155,7 +158,9 @@ <h1><span id="page_title">Flowcells</span> </h1>
                         {% set latest_running_note_key = max(k for k, v in running_notes.iteritems() if v != '') %}
                         <div class="panel panel-default running-note-panel">
                         <div class="panel-heading"><b>{{ running_notes[latest_running_note_key]['user'].encode('ascii','ignore'),latest_running_note_key.split('.')[0].encode('ascii','ignore') }}</b></div>
-                        <div class="panel-body" style="white-space: pre-line">{{ running_notes[latest_running_note_key]['note'] }}</pre></div>
+                        <div class="panel-body" style="white-space: pre-line">
+                           <script>document.write(make_markdown(`{{ running_notes[latest_running_note_key]['note'] }}`));</script></pre>
```

Comment:
I might be picky here, but is there a reason to use the ` character and not ' or "?